src/ESP8266Spiram.cpp

```diff
@@ -28,34 +28,51 @@ ESP8266Spiram Spiram;
 
 // setup of C/S line as per HSPI default
 ESP8266Spiram::ESP8266Spiram() {
```

Comment:
I thought you needed to change the CS pin for some uses, no?

In that case, wouldn't it make sense to have a constructor "ESP8266Spiram::ESP8266Spiram(int cs) {}" and  "ESP8266Spiram::ESP8266Spiram(int cs, int clockspeedhz) {}"?

Reply:
Exact ;)

Reply:
Fixed ! 

Thanks for your review :)